profile.md

```diff
@@ -486,10 +486,14 @@ Examples values of the `scope` claim:
 
 ### Group Based Authorization: wlcg.groups 
 
-Authorization may be based on the `wlcg.groups` claim. The value of the `wlcg.groups` claim is an ordered JSON array of case-sensitive strings reflecting the VO groups of which the token subject is a member. 
 
-`wlcg.groups `semantics are equivalent to existing VOMS groups. VOMS roles should be considered as optional (i.e. returned only if requested) `wlcg.groups`; selection of optional groups is discussed below.
+Authorization may be based on the `wlcg.groups` claim. The value of the `wlcg.groups` claim is an ordered JSON array of case-sensitive strings reflecting the assertion, by the issuer, of group membership of the subject of the token.
 
+Relying parties MUST only consider the group membership asserted in the token for authorization decisions.  Particularly, membership in a child group (`/cms/uscms`) does not implicitly imply membership in the parent group (`/cms`) unless the parent group is explicitly listed in the token.
```

Comment:
> Relying parties MUST only consider the group membership asserted in the token for authorization decisions

This sentence seems to rule out AuthZ decisions based on "scope"-claim values ("MUST only consider ... for authz decisions")

I think what you meant to say is something like:

    Relying parties MAY consider the subject as being a member of the various groups asserted by the
    token; however, such relying parties MUST NOT infer membership of any other groups that are not
    asserted by the OP.

Like this, I don't think you need the specific example.

Reply:
Reformulating that a bit you can keep the original meaning (i.e. keeping the authZ decisions), but using your negative (MUST NOT) which is indeed better:
```
For authorization decisions relying parties MUST NOT consider any group memberships that are not asserted by the OP in the token.
```